Demo/Demo/Models/SetupTokenRequest.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import CorePayments
+
+struct SetUpTokenRequest: APIRequest {
```

Comment:
Let's avoid the Demo using `APIRequest`. This is only internal to the SDK and not meant for merchant consumption.

Reply:
See #178

Reply:
Addressed in this commit : 3e70de279447f2684499a2f714fb330bf841946e